src/main/jbake/content/models.ad

```diff
@@ -21,180 +21,689 @@
 :jbake-tags: models
 :jbake-status: published
 :idprefix:
+:toc: preamble
+:toc-title: Content
 
-Use the links in the table below to download the pre-trained models for the Apache OpenNLP.
+The Apache OpenNLP project provides several pre-trained model files:
 
+* 1 generic model to conduct _language detection_ on a specified text input
+* 23 language specific models to accomplish: _sentence detection_, _part of speech tagging_, and _tokenization_.
+
+== Models
 IMPORTANT: All models are zip compressed (like a jar file), they *must not* be uncompressed.
 
+Use the URLs in the tables below to download the pre-trained models for use with the Apache OpenNLP toolkit.
+
+=== Language detection
+
 [options="header"]
 |======
-| Component | Language | Compatibility | Description | README and Reports | File | Signatures
-
-| Language Detector
+| Language | Description | Compatibility | README and Reports | File | Signatures
```

Comment:
Hashes and ASC signatures need to be direct links and not use the CDN via closer.cgi , i.e. replace with https://dist.apache.org/repos/dist/release/opennlp/models/ud-models-1.1/*.sha512 and https://dist.apache.org/repos/dist/release/opennlp/models/ud-models-1.1/*.asc )

Reply:
Nice finding. Note: Current live version of the site is using the same before since many, many years.
Please, drop an URL here for reference, where the above requirement is stated / documented.

Will change it accordingly, as suggested.

Reply:
https://infra.apache.org/release-distribution.html#release-content

Think we need to use https://downloads.apache.org/opennlp/models/ud-models-1.1/ as root url for hashes / sigs (TIL)

Reply:
@rzo1 I introduced the new property `opennlp.downloads` in `jbake.properties` which is resolved as configured during site build and now points to the URL you suggested.

Should be fine and ready for rollout now.